### DISCUSSION TITLE
As a solo dev, the homepage feed doesn't feel useful

### DISCUSSION THREAD

**Discussion by Chalks** [community/community#96991] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I'm mostly building things for myself by myself. I don't particularly care about random repos across github. I don't care if someone starred, sponsored, released, or any other thing **anywhere** in the github-sphere. I only care about things that are happening in repos specifically owned by me and getting to those repos quickly.

I thought that this setting would show what's going on in my repositories, but as you can see there's just a blank screen:

<img width="952" alt="Screenshot 2024-01-21 at 10 15 43 AM" src="https://github.com/community/community/assets/6187555/c54a9926-f7cc-40ce-97e2-d5acdca39efa">

I was hoping to find this (found on my public profile):

<img width="766" alt="Screenshot 2024-01-21 at 10 27 14 AM" src="https://github.com/community/community/assets/6187555/ce9ec0d1-cd5a-43b0-b39b-9cf6ae823d68">

Personally, I would like the homepage "feed" to do two things:

* help me remember where I left off last time I was here
* give me one-click access to my content

I have less than zero desire to see any sort of social-github-ness on my homepage.

**Comment by 90PabloRomero** [comment]
as someone working with people that works closed source, is totally useless, tho i watch everyday this page, go directly to the sidebar, only 4 options on the many repositories on my team going on

**Comment by bcparkison** [comment]
Agree completely with this feedback.  I don't want going to GitHub to be like going to LinkedIn.  I want to find my stuff.  This isn't a social experience.  I'm doing work, either for my job or myself.

**Comment by ghost** [comment]
I left on codeberg for this very reason: too much social crap instead of work.

**Comment by djthorpe** [comment]
I'd be keen to see a more customizable experience for the feed, so that it can be both "where I left off" and also balance that with activity on my code and repos I care about. For example,

For me and for organizations I am part of, or repos I am watching....
* Any open pull requests that need reviewed
* Recent failed github actions jobs
* Anything in my project "TODO" list that I am working on
* Issues that are assigned to me which are open (and perhaps, just been opened)
* New releases on repos 
* New or updated packages
* ...anything else?

Social:
* New stars
* Recent forks 
* New discussions or new threads in existing discussions
* New followers
* ....etc

Could it be set up so both are tracked?
